components/x-teaser/src/Status.jsx

```diff
@@ -2,14 +2,17 @@ import { h } from '@financial-times/x-engine'
 import TimeStamp from './TimeStamp'
 import RelativeTime from './RelativeTime'
 import LiveBlogStatus from './LiveBlogStatus'
+import AlwaysShowTimestamp from './AlwaysShowTimestamp'
 
 export default (props) => {
 	if (props.status) {
 		return <LiveBlogStatus {...props} />
 	}
 
 	if (props.publishedDate) {
-		if (props.useRelativeTime) {
+		if (props.alwaysDisplayTimestamp) {
+			return <AlwaysShowTimestamp {...props} />
+		} else if (props.useRelativeTime) {
```

Comment:
**suggestion**: it would be good to add this variant to Storybook & the snapshot tests somewhere, i can't remember how to do that though 😬